libs/labelbox/tests/data/annotation_import/conftest.py

```diff
@@ -730,6 +802,41 @@ def _create_prompt_response_project(
     return prompt_response_project, ontology
 
 
+def _create_offline_mmc_project(client: Client, rand_gen, data_row_json, normalized_ontology) -> Tuple[Project, Ontology, Dataset]:
```

Comment:
Why is this not a fixture? How do you intend to clean resources after the test if it is not a fixture?

Reply:
This is done in the same fashion as the other existing `_create_*_project` functions. They are used by `configured_project` fixture which does the cleaning.